src/_posts/platform/deployment/2000-01-01-deploy-java-jar-war.md

```diff
@@ -93,22 +93,17 @@ $ scalingo --app my-app env-set JAVA_VERSION=1.9
 
 ### Webapp Runner / Tomcat version selection
 
-By default, Tomcat is installed with the last version of the webapp-runner. If
-you want to use another version, you can defined the environment variable
-`JAVA_WEBAPP_RUNNER_VERSION`. The available versions are:
-
-- `8.5.11.3`
-- `8.5.51.0`
-- `9.0.52.1`
-- `9.0.68.1`
-- `9.0.80.0`
-- `9.0.83.0` (default)
-
-The 8.5.x versions are installing Tomcat 8, and the 9.0.x releases are
-installing Tomcat 9.0:
+By default, the latest `9.0.x` version of Tomcat is installed. If you want to
+use another version, you can define the environment variable
+`JAVA_WEBAPP_RUNNER_VERSION`, like so:
 
 ```sh
-# Install Tomcat 8
-
-$ scalingo env-set JAVA_WEBAPP_RUNNER_VERSION=8.5.11.3
+scalingo --app my-app env-set JAVA_WEBAPP_RUNNER_VERSION=10.1.16.0
 ```
+
+The available versions currently are:
+
+| Tomcat Version | Latest version | Notes   |
+| -------------: | -------------- | ------- |
+| `9.0`          | `9.0.83.1`     | default |
```

Comment:
Don't we want to keep the `8.5.x` version? Is it end of life?

Don't we want to keep the full list of versions? I mean, we are kind of lying here. It is stated "The available versions currently are". But there are more versions currently available (e.g. 9.0.68.1), aren't they?

Reply:
Apache Tomcat will reach EOL the 31st of March 2024.

But Heroku doesn't maintain this branch anymore. The latest version of this branch we support is `8.5.51.0`, which has been released the 11th of February 2020.
I don't think it makes sense to still have this version in our doc, but maybe you want to keep it until April?

As for the different versions of the `9.0` branch, I don't see the point to advertise all the versions we have. People should use the latest one (as per security), don't they?

Reply:
Also, according to [this page](https://central.sonatype.com/artifact/com.heroku/webapp-runner/versions), we could probably add support for `8.5` up to `8.5.68.1` (2021-12-10).

Reply:
Alright, I put `8.5` back along with a discouraging message in https://github.com/Scalingo/documentation/pull/2437/commits/138c9e2992802c01da9f5ac8d97b53737093cc96, like we do with PHP.

I'll let you tell me if I should:
- add the `8.5.68.1` version
- add all the missing versions for the `9.0.x` branch
- put all `9.0.x` versions that we have in the doc instead of only the latest

Reply:
> Also, according to [this page](https://central.sonatype.com/artifact/com.heroku/webapp-runner/versions), we could probably add support for 8.5 up to 8.5.68.1 (2021-12-10).

Did you try to check if it works to deploy a 8.5.68.1? If it does, I would add it to our list of supported versions.